controllers/users.controllers.js

```diff
@@ -0,0 +1,13 @@
+const { selectAllUsers } = require('../models/users.models');
+
+const getAllUsers = (req, res, next) => {
+  selectAllUsers()
+    .then((data) => {
+      res.status(200).send({ users: data });
```

Comment:
If you want to shorten this slightly , you can declare the `data` parameter as `users` instead, which allows you to write `.send({ users });` and javascript will understand that you want users on a key of users